src/test/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiServiceSpec.scala

```diff
@@ -437,6 +437,18 @@ class MethodConfigApiServiceSpec extends ApiServiceSpec {
       }
   }
 
+  it should "return 200 getting method inputs and outputs" in withTestDataApiServices { services =>
```

Comment:
similar to   it should "return 404 when generating a method config template from a bogus method" in withTestDataApiServices { services =>